src/content/docs/codestream/start-here/new-relic-codestream-quick-start.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: New Relic and CodeStream quick start
+metaDescription: "The quickest way to get started with CodeStream and New Relic."
+---
```

Comment:
In my mind there are 3 scenarios here: 

1.  User already has CodeStream.  In this case they can update to the latest version of CodeStream and then follow the directions in the observability tab which will look something like this mockup:
<img width="484" alt="Screen Shot 2021-10-09 at 3 01 30 PM" src="https://user-images.githubusercontent.com/47154052/137250733-94081bcc-5ac8-4c3e-b78d-2f67ff66e301.png">
 2.  User had New Relic but not codestream.  In this case I think the best thing for them is to use the CodeStream quick start which will be available from the New Relic I/O section:
<img width="232" alt="Screen Shot 2021-10-13 at 9 13 50 PM" src="https://user-images.githubusercontent.com/47154052/137250838-e0ebb79d-a833-4e5e-b702-7220dec1e2c6.png">
we should also include the URL
3.  User has neither:  
My understanding is that eventually there will be some kind of public version of New Relic.io that will allow a user to get signed up.  IN the mean time we can direct them to sign up for new relic account first or download codestream and use the above directions.

Reply:
1 and 2 tracked with my thinking on this. As soon as I get that I/O CodeStream URL, I'll be adding it to this doc.